### DISCUSSION TITLE
Cant redeem USDC FROM Avalanche TO SOLANA at phantom wallet

### DISCUSSION THREAD

**Discussion by CYCEvans** [wormhole-foundation/wormhole#684] [Problems / bugs] [answered]
I press "Redeem” button, every time they charge 2 or 3 fees then at last it keeps giving me error messages “Transaction was not confirmed in 30.02 seconds..."....Are u guy take this problem seriously ??

**Comment by evan-gray** [comment, marked as answer]
Hi @CYCEvans, an update to wormholebridge.com was deployed last night which should help with this issue - where users who experience an unconfirmed transaction or other error during the redeem process will not have to start the sequence of transactions from the beginning. Unfortunately, users encounter issues with submitting Solana transactions during periods where the network suffers from low throughput. 

#677
